app/assets/javascripts/users/gardens.js

```diff
@@ -22,6 +22,8 @@ openFarmApp.controller('gardenCtrl', ['$scope', '$http', '$rootScope',
           function(success, response) {
             if(success) {
               $scope.profileUser.gardens = response;
+              console.log(response);
```

Comment:
Hey @TheChirpyWitch Can you clean up these log statements? Or do you need them for debugging on the server?